v2x_ros_driver/include/v2x_ros_driver/v2x_radio_client.h

```diff
@@ -147,9 +147,9 @@ class V2XRadioClient
      * @brief Final check to ensure a found BSM PSID (0x0020) is not actually a PSM DSRCmsgID (0x0020). This is done by looking for the Element ID (0x0380),
```

Comment:
@jwillmartin referencing the wave.json file, the BSM PSID is 0x0020, but the PSM DSRCmsgID is 32. Should this comment be updated?